common/common.go

```diff
@@ -31,6 +31,10 @@ const (
 	DefaultConsoleVersion = "v0.1.0"
 	// Default console plugin installation OCP version
 	DefaultDynamicPluginStartOCPVersion = "4.15.0"
+	// ArgoCDLabelSelector is an env variable used for filtering ArgoCD instances to be reconciled.
+	ArgoCDLabelSelector = "ARGOCD_LABEL_SELECTOR"
+	// DefaultArgoCDLabelSelector is used when the ARGOCD_LABEL_SELECTOR is not specified or is empty.
+	DefaultArgoCDLabelSelector = ""
```

Comment:
I'm wondering if we need to re-define these here, or if we can reuse them from argocd-operator's common package itself
@raghavi101 @anandf